src/main/java/se/liu/ida/hefquin/cli/modules/ModLPG2RDFConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package se.liu.ida.hefquin.cli.modules;
+
+import org.apache.jena.cmd.ArgDecl;
+import org.apache.jena.cmd.CmdArgModule;
+import org.apache.jena.cmd.CmdGeneral;
+import org.apache.jena.cmd.ModBase;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.LPG2RDFConfiguration;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.LPG2RDFConfigurationReader;
+
+public class ModLPG2RDFConfiguration extends ModBase {
+    protected final ArgDecl lpg2RDFDescrDecl   = new ArgDecl(ArgDecl.HasValue, "LPG2RDFConfigurationDescription", "lpg2rdf");
+
+    protected LPG2RDFConfiguration cat = null;
+
+    @Override
+    public void registerWith( final CmdGeneral cmdLine ) {
+        cmdLine.getUsage().startCategory("LPG2RDFConfigurationDescription");
+        cmdLine.add(lpg2RDFDescrDecl,  "--LPG2RDFConfigurationDescription",  "file with an RDF description of the LPG2RDFConfiguration");
```

Comment:
Making `--LPG2RDFConfigurationDescription` the default name of the argument is inconveniently long. Better `--lpg2rdfconf`
```suggestion
    protected final ArgDecl lpg2RDFDescrDecl   = new ArgDecl(ArgDecl.HasValue, "lpg2rdfconf", "lpg2rdf", "LPG2RDFConfigurationDescription");

    protected LPG2RDFConfiguration cat = null;

    @Override
    public void registerWith( final CmdGeneral cmdLine ) {
        cmdLine.getUsage().startCategory("LPG to RDF Mapping");
        cmdLine.add(lpg2RDFDescrDecl,  "--lpg2rdfconf",  "file with an RDF description of the LPG-to-RDF configuration");
```